### PR TITLE
Uml 3314 content update on lpa failure page

### PR DESCRIPTION
**Pull request by allenannom**
# Purpose

Updated content on "Could not find an LPA" page - https://opgtransform.atlassian.net/browse/UML-3314

## Approach

Updated HTML and translation 

## Checklist

* [x] I have performed a self-review of my own code
* [ ] I have added relevant logging with appropriate levels to my code
* [ ] New event_codes have been documented on the [wiki page](https://opgtransform.atlassian.net/wiki/spaces/LSML2/pages/3277881441/Understanding+the+event+logs)
* [ ] I have updated documentation (Confluence/GitHub wiki/tech debt doc) where relevant
* [ ] I have added tests to prove my work
* [ ] I have added welsh translation tags and updated translation files
* [ ] I have run an accessibility tool on any pages I have made changes to and fixed any issues found
* [ ] I have notified the Interaction Designer of any content changes so that appropriate screenshots/flow diagram changes can be made
* [ ] The product team have tested these changes

### CODE REVIEW

**Review by allenannom — COMMENTED** [review]
Updated HTML and translation

File: service-front/app/features/context/UI/LpaContext.php [hunk 1]
```diff
@@ -3224,4 +3224,21 @@ public function iClickTheCancelLink(): void
...
         $this->ui->clickLink('Cancel');
     }
+    
+    /**
+     * @Given /^I click Back to your LPAs$/
+     */
+    public function iClickBackToYourLPAsLink(): void
+    {
+        $this->ui->clickLink('Back to your LPAs');
+        $this->ui->visit('/lpa/dashboard');
```
Comment (cooperaj) [inline service-front/app/features/context/UI/LpaContext.php:3234]: This visit shouldn't be required as you've clicked the link?

File: service-front/app/src/Actor/templates/actor/lpa-not-found.html.twig [hunk 2]
```diff
@@ -5,42 +5,83 @@
...
+
+                        <li>
+                            {% trans %}LPA reference number you entered is incorrect or for a different LPA{% endtrans %}
+                        </li>
+                    </ul>
+
+                    <table class="govuk-table">
+                        <caption class="govuk-table__caption">
+                            <h3 class="govuk-heading-m govuk-!-margin-bottom-2 govuk-!-margin-top-4">
+                                {% trans %} Your answers {% endtrans %}
```
Comment (cooperaj) [inline service-front/app/src/Actor/templates/actor/lpa-not-found.html.twig:37]:
```suggestion
                                {% trans %}Your answers{% endtrans %}
```

File: service-front/app/src/Actor/templates/actor/lpa-not-found.html.twig [hunk 3]
```diff
@@ -5,42 +5,83 @@
...
+                                <td class="govuk-table__cell">{{ referenceNumber }}</td>
+                            </tr>
+                        </tbody>
+                    </table>
+
+                    <p class="govuk-body">{% trans %}If you’re sure you entered the correct information, contact us.{% endtrans %}</p>
+
+                    <h3 class="govuk-heading-s govuk-!-margin-bottom-0 govuk-!-margin-top-4">{% trans %}Email{% endtrans %}</h3>
+                    <p class="govuk-body">
+                        <a class="govuk-link" href="https://www.gov.uk/call-charges">customerservice@publicguardian.gov.uk</a>
```
Comment (cooperaj) [inline service-front/app/src/Actor/templates/actor/lpa-not-found.html.twig:63]: Should this not be a mailto link?

File: service-front/app/features/context/UI/LpaContext.php [hunk 4]
```diff
@@ -3224,4 +3224,20 @@ public function iClickTheCancelLink(): void
     {
         $this->ui->clickLink('Cancel');
     }
+
+    /**
+     * @Given /^I click Back to your LPAs$/
```
Comment (cooperaj) [inline service-front/app/features/context/UI/LpaContext.php:3229]: Minor point and it doesn't really affect the running but these are both defined as 'When' steps in the feature files so from a future us point of view it'd probably be better to keep that consistent here.

**Review by cooperaj — DISMISSED** [review]
LGTM

**Review by nickdavis2001 — APPROVED** [review]

**Review by nickdavis2001 — APPROVED** [review]

**Pull request merged**